commons/src/main/scala/dagr/commons/util/StringUtil.scala

```diff
@@ -117,6 +120,9 @@ object StringUtil {
     builder.toString()
   }
 
+  /** Resolves a path from a String, and then makes the path absolute. Prefer this to PathUtil.pathTo elsewhere. */
+  def pathTo(first: String, more: String*): Path = Paths.get(first, more:_*).toAbsolutePath.normalize
```

Comment:
I was already to fall on my sword and take the blame for this - but then I looked and it's not in `StringUtil` on master...